platform/kube/client.go

```diff
@@ -261,21 +261,49 @@ func (cl *Client) Get(key model.Key) (proto.Message, bool) {
 	return out, true
 }
 
-// Put implements registry operation
-func (cl *Client) Put(k model.Key, v proto.Message) error {
-	out, err := modelToKube(cl.mapping, &k, v)
+// Post implements registry operation
+func (cl *Client) Post(key model.Key, v proto.Message) error {
+	if key.Kind == model.IngressRule {
+		return fmt.Errorf("unsupported operation: cannot put a config element of kind '%s'", key.Kind)
```

Comment:
No I meant, change the error message to "cannot create Ingress rule through third party resource config"